ipt_connect/IPTdev/models.py

```diff
@@ -38,10 +38,16 @@ def special_mean(vec):
 
 	vec = vec[nlow:nhigh]
 
-	try:
+	if len(vec):
 		return float(sum(vec)) / len(vec)
-	except ZeroDivisionError:
-		return 0
+	return 0
+
+def iypt_mean(vec):
```

Comment:
Why implementing iypt_mean here? Is that used in national tournaments?

Reply:
Yes, we tried to use `ipt_connect` on sub-sublocal stage of IYPT (Repnoe). I cannot claim that the experiment was 100% successful, but we've made some onclusions and applied some patches.